dask/distributed/__init__.py

```diff
@@ -2,3 +2,10 @@
 from .scheduler import Scheduler
 from .client import Client
 from .ipython_utils import dask_client_from_ipclient
+
+from warnings import warn
+
+warn(FutureWarning('''
```

Comment:
Yeah, that's tricky. For some data points, SymPy subclasses DeprecationWarning, and turns on visibility for those. Bokeh raises a `UserWarning` when someone uses the deprecated `_legacy_charts`. Probably not worth spending a lot of time figuring this out, I'd just go with something. What you have here might even be fine.